example.danra.yaml

```diff
@@ -73,6 +73,32 @@ inputs:
         name_format: "{var_name}"
     target_output_variable: forcing
 
+  danra_derived_forcings:
+    path: https://mllam-test-data.s3.eu-north-1.amazonaws.com/single_levels.zarr
+    dims: [time, x, y]
+    derived_variables:
+      toa_radiation:
+        kwargs:
+          time: time
+          lat: lat
+          lon: lon
+        function: mllam_data_prep.derived_variables.calculate_toa_radiation
+      hour_of_day:
+        kwargs:
+          time: time
+        function: mllam_data_prep.derived_variables.calculate_hour_of_day
+    dim_mapping:
+      time:
+        method: rename
```

Comment:
In principle we don't since they have the same name, but as it is currently implemented all the dimensions in the output must be given in the `dim_mapping` part of the config, and there isn't an option for "this name of this dimension is unchanged". You are welcome to add that to the readme if you like :) I just did it like this to be explicit rather than implicit about what is going on.